pkg/packagekit/autoupdates.jsx

```diff
@@ -249,67 +248,128 @@ function getBackend(forceReinit) {
  * Properties:
  * onInitialized(enabled): (optional): callback once backend knowsn whether automatic updates are enabled
  */
-export default class AutoUpdates extends React.Component {
+export const AutoUpdatesBody = (props) => {
+    const { enabled, type, day, time } = props;
+    const days = {
+        "": "everyday",
+        mon: "every Monday",
+        tue: "every Tuesday",
+        wed: "every Wednesday",
+        thu: "every Thursday",
+        fri: "every Friday",
+        sat: "every Saturday",
+        sun: "every Sunday"
+    };
+
+    let str;
+    if (enabled) {
+        str = type == "security" ? _("Security updates ") : _("Updates ");
+        str += day == "" ? cockpit.format(_("will be applied $0 at $1"), days[day], time) : cockpit.format(_("will be applied $0 at $1"), days[day], time);
```

Comment:
Whats the difference between the first and the second string?

Reply:
Oh, they were different at first. Then I ammended them without realizing both strings are the same now. Will fix it